docs/resources/instance_user_data.md

```diff
@@ -0,0 +1,77 @@
+---
+page_title: "Scaleway: scaleway_instance_user_data"
+description: |-
+Manages Scaleway Compute Instance User Data.
+---
+
+# scaleway_instance_server
+
+Creates and manages Scaleway Compute Instance User Data values.
+
+User data is a key value store API you can use to provide data from and to your server without authentication. It is the mechanism by which a user can pass information contained in a local file to an Instance at launch time.
+
+The typical use case is to pass something like a shell script or a configuration file as user data.
```

Comment:
such as cloud-init for instance

Reply:
Yes, I've taken some documentation already present on the resource instance server